tv.kodi.Kodi.yml

```diff
@@ -116,39 +118,42 @@ modules:
     sources:
       - type: git
         url: https://github.com/xbmc/xbmc.git
-        commit: 59d6be57497667de7d40aacf066be7875a6a94b4
+        commit: 5d18c0baab1c7ecec354d92eadcabbb4b3ea280c
       - type: patch
         path: kodi.sh.in.patch
+        use-git: true
+      - type: patch
+        path: metainfo.patch
+        use-git: true
       - type: file
         path: addon-list.txt
         dest: build/download/
-        dest-filename: addon-list.txt
       - type: file
         path: tools/patch-addon-manifest.sh
         dest: build/download/
       - type: shell
         commands:
           - build/download/patch-addon-manifest.sh build/download/addon-list.txt
-      - type: patch
-        path: addons-no-startupenable.patch
+          - git add system/addon-manifest.xml
+          - git commit system/addon-manifest.xml -m "[patch] make addons optional"
```

Comment:
Why are we doing this?

Reply:
I'd like to get rid of the "dirty" in version info, but looks like it didn't work